### PR TITLE
feat(precinct-scanner): Show message when card inserted backwards

### PR DESCRIPTION
**Pull request by jonahkagan**
- Update the useSmartcard hook to consume the new smartcard status
- Use the status to determine when to show a new screen with a message about the card being backwards

TODO: write tests

### CODE REVIEW

File: libs/utils/src/types.ts [hunk 1]
```diff
@@ -97,26 +97,27 @@ export interface Storage {
   clear(): Promise<void>;
 }
 
-export interface CardAbsentApi {
-  present: false;
+export interface CardApiNotReady {
+  status: 'no_card' | 'error';
```
Comment (benadida) [inline libs/utils/src/types.ts:101]: suggest something slightly more descriptive than `error`, something specific to the card (as opposed to, say, the reader itself, which we might want to add later.) Something like `error_card_unreadable` or something like it.

**Review by benadida — APPROVED** [review]
one small suggestion but overall this is the right path.

File: libs/utils/src/types.ts (hunk 1 above)
Comment (jonahkagan) [reply libs/utils/src/types.ts:101]: I was thinking we could add an `errorType` field if we wanted to surface multiple errors later.

Comment (benadida) [reply libs/utils/src/types.ts:101]: I can live with that!

**Pull request merged**